src/components/Grid.vue

```diff
@@ -60,11 +61,11 @@ export default {
       const selectedRowIds = selectedNodes.map((item) => item.data.id);
       emit("selection-changed", selectedRowIds);
     };
-    // Using props.selection from the parent component,
+    // Fetching the data from IndexedDB,
```

Comment:
revert this comment to what it used to be `// Using props.selection from the parent component,`

Reply:
this should have the complete comment that used to be here. Nothing has changed in this PR shoudl it shoud say

    // Using props.selection from the parent component,
    // find the row id in the grid that matches the row id
    // in indexedDB that has the selected value = true,
    // then set that matching row in the grid to selected